src/Symfony/Component/Notifier/Bridge/Telegram/TelegramTransport.php

```diff
@@ -82,7 +82,7 @@ protected function doSend(MessageInterface $message): SentMessage
 
         if (!isset($options['parse_mode']) || TelegramOptions::PARSE_MODE_MARKDOWN_V2 === $options['parse_mode']) {
             $options['parse_mode'] = TelegramOptions::PARSE_MODE_MARKDOWN_V2;
-            $options['text'] = str_replace('.', '\.', $message->getSubject());
+            $options['text'] = preg_replace('/([\[_\*\[\]\(\)~`>#\+-=\|{}\.!])/m', '\\\\$1', $message->getSubject());
```

Comment:
`\[` is referenced twice.
IIRC you don't have to escape everything
the `m` could be removed

Reply:
Oh, indeed, I will remove it.